### PR TITLE
PLYLoader: Honor different attribute names.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: see #22675

**Description**

Update the original PR with https://github.com/mrdoob/three.js/pull/22675#issuecomment-945169422.

### CODE REVIEW

File: examples/jsm/loaders/PLYLoader.js [hunk 1]
```diff
@@ -363,25 +363,51 @@ class PLYLoader extends Loader {
 
 		function handleElement( buffer, elementName, element ) {
 
+			function findAttrName( names ) {
+
+				for ( let i = 0, l = names.length; i < l; i ++ ) {
```
Comment (Mugen87) [inline examples/jsm/loaders/PLYLoader.js:368]: I've notice that a major part of the performance drop was introduced by the for/in loop. Using a canonical one makes it much faster.

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev9/examples/webgl_loader_ply.html

File: examples/jsm/loaders/PLYLoader.js (hunk 1 above)
Comment (mrdoob) [reply examples/jsm/loaders/PLYLoader.js:368]: Things never change... 😁

I suspect it's okay that is a bit slower.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**